src/components/AccountLoginForm.jsx

```diff
@@ -23,7 +23,7 @@ const probableLoginFieldNames = [
 
 const renderers = {
   password: ({ t }) => <PasswordField noAutoFill />,
-  date: () => <Field type="date" />,
+  date: () => <Field type="date" placeholder="YYYY-MM-DD" />,
```

Comment:
You may want to translate it?

Reply:
It's complicated to translate it as we are expecting an unique format.

Reply:
I was talking about `AAAA-MM-JJ` for french, but it's not necessary for now I think 👍